deploy/cht_sync/values.yaml.template

```diff
@@ -0,0 +1,25 @@
+
+postgres:
+  enabled: true  # if true a postgres service will not be created in this cluster
+  user: "postgres"
```

Comment:
Can we leave this blank, so no users will ever end up using these default values?